### ISSUE TITLE
Configure step doesn't check for libicu

### ISSUE THREAD

**Issue by forslund** [MycroftAI/mimic#89]
When building mimic on a system without `libicu` the configuration step finishes without issues but the make step fails when trying to build objects requiring `libicu`

**Comment by forslund** [comment]
I've been looking into this and it seem the check is there but it takes no action. See https://github.com/MycroftAI/mimic/blob/development/configure.ac#L76

My suggestion is that this line is changed to
```
PKG_CHECK_MODULES([ICU], [icu-i18n],
                  [m4_ignore],
                  [AC_MSG_ERROR([Unable to find an installation of ICU])])
```

@zeehio, can I go ahead with this change or is there something I'm missing?

**Comment by forslund** [comment]
Resolved by PR #92
